js/templates/modals/settings/addressesForm.html

```diff
@@ -1,73 +1,73 @@
-<form class="box padKids padStack rowMd clrP border clrBr js-addAddressForm">
+<form class="box padKids padStack contentBox pad rowMd clrP border clrBr js-addAddressForm">
   <div class="flexRow">
     <div class="col3">
-      <label for="settingsAddressName"><%= ob.polyT('settings.addresses.name') %></label>
+      <label for="settingsAddressName" class="required"><%= ob.polyT('settings.addresses.name') %></label>
     </div>
-    <div class="col9">
+    <div class="col6">
       <% if (ob.errors.name) print(ob.formErrorTmpl({ errors: ob.errors.name })) %>
-      <input type="text" class="clrBr" name="name" id="settingsAddressName" value="<%= ob.name %>" />
+      <input type="text" class="clrBr clrSh2" name="name" id="settingsAddressName" value="<%= ob.name %>" placeholder="<%= ob.polyT('settings.addressesTab.placeholderName') %>" />
     </div>
   </div>
   <div class="flexRow">
     <div class="col3">
       <label for="settingsAddressCompany"><%= ob.polyT('settings.addresses.company') %></label>
     </div>
-    <div class="col9">
+    <div class="col6">
       <% if (ob.errors.company) print(ob.formErrorTmpl({ errors: ob.errors.company })) %>
-      <input type="text" class="clrBr" name="company" id="settingsAddressCompany" value="<%= ob.company %>" />
+      <input type="text" class="clrBr clrSh2" name="company" id="settingsAddressCompany" value="<%= ob.company %>" placeholder="<%= ob.polyT('settings.optional') %>" />
     </div>
   </div>
   <div class="flexRow">
     <div class="col3">
       <label for="settingsAddressLineOne"><%= ob.polyT('settings.addresses.street') %></label>
     </div>
-    <div class="col9">
+    <div class="col6">
       <% if (ob.errors.addressLineOne) print(ob.formErrorTmpl({ errors: ob.errors.addressLineOne })) %>
-      <input type="text" class="clrBr" name="addressLineOne" id="settingsAddressLineOne" value="<%= ob.addressLineOne %>" />
+      <input type="text" class="clrBr clrSh2" name="addressLineOne" id="settingsAddressLineOne" value="<%= ob.addressLineOne %>"  placeholder="<%= ob.polyT('settings.addressesTab.placeholderAddress') %>" />
     </div>
   </div>
   <div class="flexRow">
     <div class="col3">
       <label for="settingsAddressLineTwo"><%= ob.polyT('settings.addresses.apt') %></label>
     </div>
-    <div class="col9">
+    <div class="col3">
       <% if (ob.errors.addressLineTwo) print(ob.formErrorTmpl({ errors: ob.errors.addressLineTwo })) %>
-      <input type="text" class="clrBr" name="addressLineTwo" id="settingsAddressLineTwo" value="<%= ob.addressLineTwo %>" />
+      <input type="text" class="clrBr clrSh2" name="addressLineTwo" id="settingsAddressLineTwo" value="<%= ob.addressLineTwo %>" placeholder="<%= ob.polyT('settings.optional') %>" />
     </div>
   </div>
   <div class="flexRow">
     <div class="col3">
       <label for="settingsAddressCity"><%= ob.polyT('settings.addresses.city') %></label>
     </div>
-    <div class="col9">
+    <div class="col6">
```

Comment:
well, it does shorten the space for the error. So, if it's a longer one (you could fudge a long one in the dom inspector), it will be limited to the width of the field and then wrap. Which... could very well be what we want.